src/resolve-host/resolve-host.c

```diff
@@ -64,10 +64,12 @@ static void print_source(uint64_t flags, usec_t rtt) {
         fputs("\n-- Information acquired via", stdout);
 
         if (flags != 0)
-                printf(" protocol%s%s%s",
+                printf(" protocol%s%s%s%s%s",
```

Comment:
I know it's not your changes introducing this, but what is the deal with the mixing of fputs/fputc/printf in this function?  Why aren't the fputc and fputs instances using putc and puts since they're all using stdout anyways?  If there's a valid reason, why isn't it applicable to the printf() calls rendering them fprintf(stdout, ..) ?  Just looks arbitrarily inconsistent to this reader.